crates/neon-runtime/src/napi/call.rs

```diff
@@ -28,7 +29,19 @@ pub unsafe extern "C" fn get_isolate(_info: FunctionCallbackInfo) -> Env { unimp
 // FIXME: Remove. This will never be implemented
 pub unsafe extern "C" fn current_isolate() -> Env { panic!("current_isolate won't be implemented in n-api") }
 
-pub unsafe extern "C" fn is_construct(_info: FunctionCallbackInfo) -> bool { unimplemented!() }
+pub unsafe extern "C" fn is_construct(env: Env, info: FunctionCallbackInfo) -> bool {
+    let mut target: MaybeUninit<napi::napi_value> = MaybeUninit::zeroed();
+    println!("CRASHING HERE");
+    let status = napi::napi_get_new_target(
```

Comment:
@kjvalencik @goto-bus-stop Can you see anything obvious here I might be doing wrong? For some reason, `cargo test napi -- --nocapture` prints `CRASHING HERE` and then silently dies.